ivy/stateful/activations.py

```diff
@@ -170,10 +170,26 @@ def _forward(self, x):
 
 
 class Softmax(Module):
-    def __init__(self, axis: int = -1):
-        """Apply the SOFTMAX activation function."""
+    def __init__(
+        self,
+        axis: int = -1,
+        complex_mode: Literal["split", "magnitude", "jax"] = "jax",
+    ):
+        """
+        Apply the SOFTMAX activation function.
+
+        Parameters
+        ----------
+        axis
+            The axis which we apply softmax op on.
+        complex_mode
+            Specifies how to handle complex input. See
+            ``ivy.func_wrapper.handle_complex_input`` for more detail.
+        """
+
         Module.__init__(self)
         self._axis = axis
```

Comment:
The blank line (line 189) is causing linting errros
```suggestion
        """
        Apply the SOFTMAX activation function.

        Parameters
        ----------
        axis
            The axis which we apply softmax op on.
        complex_mode
            Specifies how to handle complex input. See
            ``ivy.func_wrapper.handle_complex_input`` for more detail.
        """
        Module.__init__(self)
        self._axis = axis
```